README.md

```diff
@@ -0,0 +1,46 @@
+# How to embed Brandgility's configurator into an iframe:
+- get the id of template to be configured from Brandgility application
+- copy and paste below code into your page/component, replace `:id` in `src` attribute with your template's id:
+```html
+<iframe
+  style={{ width: '100%', height: '100%', border: 'none' }}
+  src="<BRANDGILITY_ENDPOINT>/embedded-template-configure/{new|edit}/:id" />
+```
+
+# API library
+*Brandgility embedded api library allows you to issue commands to Brandgility configurator in embedded mode*
+
+## Methods
+
+| method | description | parameters |
+|:-----:|:-----:|:-----:|:-----:|
+| `on` | subscribes to event | eventName: <`String`>, callback: <`Function`> |
+| `emit` | triggers an event | eventName: <`String`>, argument: <`Any`> |
```

Comment:
not Any, but anything serializable by [The structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)